### PR TITLE
Replace the hard coded path with code that derives the path through a function

### PR DESCRIPTION
**Pull request by jeevithiesh**

### CODE REVIEW

File: goa/Classes/ext/LipiTKNode.cpp [hunk 1]
```diff
@@ -90,7 +90,10 @@ bool LipiTKNode::initialize(int width, int height, Point position) {
     
         #elif (CC_TARGET_PLATFORM == CC_PLATFORM_ANDROID)
             //initialize lipiTK
-             path = "/storage/emulated/0/Android/data/com.maq.xprize.chimple.hindi/files/res/android-lipitk";
+                std::string fileName = "res/android-lipitk/projects/lipiengine.cfg";
+                std::string path1 = FileUtils::getInstance()->fullPathForFilename(fileName);
+                int length = path1.length();
+                path = path1.substr(0, path1.length() - 24);
```
Comment (ankitk20) [inline goa/Classes/ext/LipiTKNode.cpp]: Add comment for this line

File: goa/Classes/ext/LipiTKNode.cpp [hunk 2]
```diff
@@ -90,7 +90,10 @@ bool LipiTKNode::initialize(int width, int height, Point position) {
     
         #elif (CC_TARGET_PLATFORM == CC_PLATFORM_ANDROID)
             //initialize lipiTK
-             path = "/storage/emulated/0/Android/data/com.maq.xprize.chimple.hindi/files/res/android-lipitk";
+                std::string fileName = "res/android-lipitk/projects/lipiengine.cfg";
+                std::string path1 = FileUtils::getInstance()->fullPathForFilename(fileName);
```
Comment (ankitk20) [inline goa/Classes/ext/LipiTKNode.cpp]: Give a good name and never use path1 or path2 or path 3 etc.

**Review by ankitk20 — CHANGES_REQUESTED** [review]

File: goa/Classes/ext/LipiTKNode.cpp [hunk 3]
```diff
@@ -90,7 +90,10 @@ bool LipiTKNode::initialize(int width, int height, Point position) {
     
         #elif (CC_TARGET_PLATFORM == CC_PLATFORM_ANDROID)
             //initialize lipiTK
-             path = "/storage/emulated/0/Android/data/com.maq.xprize.chimple.hindi/files/res/android-lipitk";
+                std::string fileName = "res/android-lipitk/projects/lipiengine.cfg";
```
Comment (ankitk20) [inline goa/Classes/ext/LipiTKNode.cpp]: string&

File: goa/Classes/ext/LipiTKNode.cpp [hunk 4]
```diff
@@ -90,7 +90,10 @@ bool LipiTKNode::initialize(int width, int height, Point position) {
...
+                std::string path1 = FileUtils::getInstance()->fullPathForFilename(fileName);
+                int length = path1.length();
```
Comment (ankitk20) [inline goa/Classes/ext/LipiTKNode.cpp]: Remove this line

**Review by ankitk20 — CHANGES_REQUESTED** [review]

File: goa/Classes/ext/LipiTKNode.cpp (hunk 1 above)
Comment (jeevithiesh) [reply goa/Classes/ext/LipiTKNode.cpp]: I have added a comment

File: goa/Classes/ext/LipiTKNode.cpp (hunk 2 above)
Comment (jeevithiesh) [reply goa/Classes/ext/LipiTKNode.cpp]: I have removed the variable path1

File: goa/Classes/ext/LipiTKNode.cpp (hunk 4 above)
Comment (jeevithiesh) [reply goa/Classes/ext/LipiTKNode.cpp]: I have removed the line

File: goa/Classes/ext/LipiTKNode.cpp (hunk 3 above)
Comment (jeevithiesh) [reply goa/Classes/ext/LipiTKNode.cpp]: I have changed the type of fileName

**Review by ankitk20 — APPROVED** [review]

**Comment by ankitk20** [comment]
Fixes #62

**Pull request merged**